<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSearchLocations">
      <attribute name="title">
       <string>My search locations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item row="1" column="0">
        <widget class="QFrame" name="frameSettingsItem">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="SettingsItem" name="settingsItem" native="true">
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFrame" name="frameButtons">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="buttonAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonDuplicate">
            <property name="text">
             <string>Duplicate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>217</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelAvailableLocations">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Available search locations:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelEditLocation">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Edit search location:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QListView" name="listViewLocations">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="buttonSetDefault">
            <property name="toolTip">
             <string>Default search location for new search tabs</string>
            </property>
            <property name="text">
             <string>Set default</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGlobalSearchLocations">
      <attribute name="title">
       <string>Global search locations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item row="2" column="0">
        <widget class="QFrame" name="frameGlobalSettingsItem">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="SettingsItem" name="globalSettingsItem" native="true">
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelAvailableGlobalLocations">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Available search locations:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QListView" name="listViewGlobalLocations">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="3">
        <widget class="QFrame" name="frame_2">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="buttonSetDefaultGlobal">
            <property name="toolTip">
             <string>Default search location for new search tabs</string>
            </property>
            <property name="text">
             <string>Set default</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>285</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelGlobalLocation">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Global search locations are defined in config.txt and can only be viewed here but not edited. To change their settings modify config.txt or UserConfig.txt.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Global search location:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOptions">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelFontFamily">
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFontComboBox" name="fontComboBox">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelFontSize">
         <property name="text">
          <string>Font size:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editFontSize">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelTabWidth">
         <property name="text">
          <string>Tab width:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editTabWidth">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelPreviewLines">
         <property name="text">
          <string>Lines to show in overview:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editPreviewLines">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="checkShowLineNumbers">
         <property name="text">
          <string>Show line numbers</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="checkMatchOverFiles">
         <property name="text">
          <string>Clicking &quot;Next match&quot; continues with next file if the last match has been reached</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="checkConfirmClose">
         <property name="text">
          <string>Ask whether to really close the application</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QCheckBox" name="checkShowMatchList">
         <property name="text">
          <string>Show match list on right side by default</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="checkActivateFirstMatch">
         <property name="text">
          <string>Automatically open first matching document after searching</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="checkDarkTheme">
         <property name="text">
          <string>Use dark theme (requires restart)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingsItem</class>
   <extends>QWidget</extends>
   <header>dialogs.SettingsItem</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>settingsItem</tabstop>
  <tabstop>buttonSetDefault</tabstop>
  <tabstop>listViewLocations</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>buttonDuplicate</tabstop>
  <tabstop>buttonRemove</tabstop>
  <tabstop>globalSettingsItem</tabstop>
  <tabstop>buttonSetDefaultGlobal</tabstop>
  <tabstop>listViewGlobalLocations</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>editFontSize</tabstop>
  <tabstop>editTabWidth</tabstop>
  <tabstop>editPreviewLines</tabstop>
  <tabstop>checkMatchOverFiles</tabstop>
  <tabstop>checkConfirmClose</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="SettingsDialog.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>addLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemove</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>removeLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>okClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDuplicate</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>duplicateLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSetDefault</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>setDefaultLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSetDefaultGlobal</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>setDefaultLocationGlobal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addLocation()</slot>
  <slot>removeLocation()</slot>
  <slot>okClicked()</slot>
  <slot>selectionChanged(QModelIndex)</slot>
  <slot>duplicateLocation()</slot>
  <slot>setDefaultLocation()</slot>
  <slot>setDefaultLocationGlobal()</slot>
 </slots>
</ui>
